### PR TITLE
WA-18217 Update Onfido API to 3.6

### PR DESCRIPTION
**Pull request by JakovVisn**

### CODE REVIEW

**Comment by Oxxyg33n** [comment]
Task linked: [WA-18217 Update Onfido API to 3.6](https://app.clickup.com/t/6633470/WA-18217)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export GOFLAGS=-mod=vendor
+export GOPRIVATE=github.com/wallester
+
+mod-vendor:
+	go mod tidy && go mod vendor
+
+validate:
+	openapi-generator validate -i /Users/jakov/go/src/github.com/wallester/go-onfido-openapi/openapi.yaml
```
Comment (SergeySaprykin) [inline Makefile:8]: I thing path not correct /Users/jakov/go/src

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+generate:
+	openapi-generator generate \
+	-i /Users/jakov/go/src/github.com/wallester/go-onfido-openapi/openapi.yaml \
```
Comment (SergeySaprykin) [inline Makefile:12]: same problem

File: Makefile (hunk 1 above)
Comment (JakovVisn) [reply Makefile:8]: Works locally)

Comment (SergeySaprykin) [reply Makefile:8]: only for you

**Review by SergeySaprykin — APPROVED** [review]

**Review by SergeySaprykin — APPROVED** [review]

**Review by Matrix278 — APPROVED** [review]

**Pull request merged**